services/API-service/src/api/event/event.service.ts

```diff
@@ -481,11 +484,11 @@ export class EventService {
         'case when event.closed = true then event."endDate" end as "endDate"',
         'disaster."mainExposureIndicator" as "exposureIndicator"',
         'event."mainExposureValue" as "exposureValue"',
-        `CASE 
-        WHEN event."forecastTrigger" = true THEN 'Trigger' 
-        WHEN event."forecastSeverity" = 1 THEN 'High warning' 
-        WHEN event."forecastSeverity" = 0.7 THEN 'Medium warning' 
-        WHEN event."forecastSeverity" = 0.3 THEN 'Low warning' 
+        `CASE
+        WHEN event."forecastTrigger" = true THEN 'Trigger'
+        WHEN event."forecastSeverity" = 1 THEN 'High warning'
```

Comment:
Hmm, this is my change, but realizing that this code was and is suboptimal, but mostly in current practice this  will go wrong because drought/flash-flood/typhoon warnings will all have forecastSeverity=1, while for floods the 'high warning' will for now not be implented (or if so, then include this in AB#33695). So for now changing this to 'Warning' to at least better reflect the current situation with this suboptimal code.